lib/ansible/modules/network/meraki/meraki_admin.py

```diff
@@ -117,6 +119,28 @@
     org_name: YourOrg
     state: absent
     email: jane@doe.com
+
+- name: Create a new administrator with full access to a tag
+  meraki_admin:
+    auth_key: abc12345
+    org_name: YourOrg
+    state: present
+    name: Jane Doe
+    orgAccess: read-only
+    email: jane@doe.com
+    tags:
+        - {"tag": "tenant", "access": "full"}
```

Comment:
This tag/access isn't defined in the description for tags. Can you add that information like you did for the json dict for networks?

Reply:
I've updated this so please take a look and let me know if what I have is more obvious.